src/Runners/PHPUnit/SuiteLoader.php

```diff
@@ -136,39 +141,29 @@ public function load(): void
             && $this->configuration !== null
             && ! $this->configuration->testSuite()->isEmpty()
         ) {
+            $testSuiteCollection = $this->configuration->testSuite()->asArray();
             $this->suitesName = array_map(static function (TestSuite $testSuite): string {
                 return $testSuite->name();
-            }, $this->configuration->testSuite()->asArray());
+            }, $testSuiteCollection);
         } elseif (
             $this->configuration !== null
             && ! $this->configuration->testSuite()->isEmpty()
         ) {
-            $testSuiteCollection = $this->configuration->testSuite()->asArray();
-            if (count($this->options->testsuite()) > 0) {
-                $suitesName = array_map(static function (TestSuite $testSuite): string {
-                    return $testSuite->name();
-                }, $testSuiteCollection);
-                foreach ($this->options->testsuite() as $testSuiteName) {
-                    if (! in_array($testSuiteName, $suitesName, true)) {
-                        throw new RuntimeException("Suite path {$testSuiteName} could not be found");
```

Comment:
PHPUnit returns success on running non-existent suite; we implement same behavior and remove this check.